docs/Converters/UKBtoBIDS.md

```diff
@@ -0,0 +1,87 @@
+<!-- markdownlint-disable MD046 -->
+# `ukb-to-bids` – Conversion of the UK Biobank (UKB) to BIDS
+
+!!! quote "Description reproduced from the [UK Biobank webpage](https://www.ukbiobank.ac.uk/)"
+    UK Biobank is a large-scale biomedical database and research resource, containing in-depth genetic and health information from half a million UK participants. The database is regularly augmented with additional data and is globally accessible to approved researchers undertaking vital research into the most common and life-threatening diseases. It is a major contributor to the advancement of modern medicine and treatment and has enabled several scientific discoveries that improve human health
+
+## Dependencies
+
+If you installed the core of Clinica, this converter needs no further dependencies.
+
+## Downloading UK Biobank
+
+The UK Biobank to BIDS converter requires the user to have downloaded the UK Biobank imaging and clinical data.
+
+Once you have access to it, the imaging data can be downloaded and extracted using this tutorial: https://biobank.ctsu.ox.ac.uk/~bbdatan/Accessing_UKB_data_v2.3.pdf, and the clinical data can be extracted using the information from this repository https://github.com/kenhanscombe/ukbtools.
+
+
+Be careful about two things: 
+    - the converter only takes in the nifti provided by UK Biobank, not the dicoms.
+    - the converter will not work if you do not have among the clinical data the year of birth, the age at recruitment and at sessions and the sex of the subject.
+
+## Supported modalities
+
+Please note that this converter processes the following modalities : 
+- T1W
+- T2 Flair
+- DWI
+- SWI
+- tfMRI
+- rsfMRI
+
+!!! Chosen files
+    Whenever possible, we use the rawest files in order not to impose opinions onto the user. When available we get the associated json.
+    - T1W -> T1.nii.gz: cropped so there is no neck and defaced. The rawest image would be the simply defaced one, but since we have no interest in the neck we don't use it. We do not want any other corrections.
+    - T2 Flair -> T2_FLAIR: cropped so there is no neck and defaced. The reasons for choosing this image are the same as for T1.
+    - DWI -> AP.nii.gz/PA.nii.gz + bvals and bvecs. It is the rawest images we can get.
+    - rsfMRI -> rfMRI.nii.gz
+    - tfMRI -> tfMRI.nii.gz or dicom equivalent to get the json associated
+    - SWI -> SWI.nii.gz which is the combined coil version. We would get a rawer version, but dcm2niix seems to have trouble handling this format, so it is simpler to go with this version. Plus, swi is not fully integrated to bids so changes may be coming, hence a simple version for now.
```

Comment:
"dcm2niix seems to have trouble handling this format" is too broad and unspecific. Reformulate.